skimage/morphology/_flood_fill.py

```diff
@@ -100,10 +105,15 @@ def flood_fill(image, seed_point, new_value, *, selem=None, connectivity=None,
            [5, 5, 5, 5, 2, 2, 5],
            [5, 5, 5, 5, 5, 5, 3]])
     """
+    if inplace is not None:
+        warn('The inplace parameter will change'
```

Comment:
```suggestion
        warn('The inplace parameter will change '
```

Reply:
Just a suggestion which may be more clearer:
```
The `inplace` parameter is depreciated and will be removed 
in version 0.19.0. Use `in_place` instead.
```
But I'm fine either.